src/into_dart.rs

```diff
@@ -253,12 +257,12 @@ macro_rules! dart_typed_data_type_trait_impl {
             #[doc(hidden)]
             #[no_mangle]
             pub(crate) unsafe extern "C" fn $free_zero_copy_buffer_func(
-                isolate_callback_data: *mut c_void,
+                _isolate_callback_data: *mut c_void,
```

Comment:
btw, curious why we cannot use isolate_callback_data today while we could before - has something changed in Dart/Rust?

Reply:
For some reason it still crashes when I use it instead of the original ptr. So I decided to put the ptr and length in the Box.